### PR TITLE
Fix up Notes Pages

### LINKED ISSUE #3173: Pull in notes layout to its own proper layout
Blocked by merge pending of #3167 but we should not have notes pages each needing to be updated. 

Once #3167 is merged, I will take a crack at merging the overlap to its own layout so in the future it would only need to be updated once and not in multiple places for changes to icons.

Using this issue to keep track of the effort until that point, so we do not forget once #3167 is merged

### PR DESCRIPTION
**Pull request by kenman345**
Closes #3173 

Found a few issues with Notes pages during these efforts.
1. the new Semantics UI version broke some spacing. I made the spacing as close to what was possible previously.
2. Less pages to go through if future revisions are made to icon naming.
3. We had a `Schwab.html` page but it was not being shown through the UI. Making the pages in the notes folder appear dynamically will prevent this from ever happening again.
4. Index.html in the notes folder is poor practice. It is instead advisable under Jekyll best practices to have it named `notes.html` and let Jekyll make it into the `index.html` in the `notes` subfolder.

As part of this PR:
1. I added an image for Schwab notes card to use. 
2. I set the default description for meta tags to use the `site.description` by default when no `page.description` exists. **This aligns the functionality with the `jekyll-seo-tag` github pages supported plugin.**
3. I set the title of the meta tags and pages to try and be the `page.title` and then default back to `site.title` when not specified for the page. **This aligns the functionality with the `jekyll-seo-tag` github pages supported plugin.**
4. I made sure that we never have notes missing from the overview page for notes. This is helpful in the future as it reduces the overhead/process to add more notes pages. 
5. Migrated overlapping html from individual notes pages to the layout where it made sense to do.
6. I added other Meta html tag changes to align with proper linking over of those tags to the page in question. Meta tags currently are all the exact same even though the content of each page is different. This change will help those be reflected to web crawlers and links over from other sites.
7. I found that we were partially using the `site.rep` variable in places. I moved those to use the variable in place of the hardcoded name.

### CODE REVIEW

**Comment by kenman345** [comment]
someone should take a look at this...

**Comment by kenman345** [comment]
No ones even reviewing this?

**Comment by kenman345** [comment]
Fixed the merge conflict, this still hasnt been reviewed though. Can this PR get some love?

File: _layouts/secondary.html
Comment (fpigerre) [inline _layouts/secondary.html]: This is great for moving the notes pages to their own layout page :+1:

**Comment by fpigerre** [comment]
Hello @kenman345!

Sorry this pull request hasn't been looked at. I'll try my best to show it some ❤️ .
So far the changes you've made look great! Awesome work with the tags.
Do you think you might be able to refactor the `notes.html` pages as a `notes/index.html` page? This makes the root of the site a lot cleaner.

Thankyou very much!
psgs 🌴

**Review by fpigerre — CHANGES_REQUESTED** [review]
* Please refactor `notes.html` as `notes/index.html` if possible

File: notes.html
Comment (fpigerre) [inline notes.html]: Beautiful 🌴

**Comment by kenman345** [comment]
Just oooking at this on my email, will fix things up tomorrow when I get
the chance. Thanks for the feedback

File: _layouts/secondary.html
Comment (kenman345) [reply _layouts/secondary.html]: Made a change to this based on moving the notes.html page to the subfolder for notes and renaming it to index.html

**Comment by kenman345** [comment]
@psgs All the changes around moving the notes.html page to be `notes/index.html` have been made. This should be ready for further review now!

**Review by fpigerre — APPROVED** [review]
Thankyou very much @kenman345! :+1:

**Comment by fpigerre** [comment]
@carlgo11 Would you like to take a look and approve this pull request?

**Comment by kenman345** [comment]
@psgs just noting contrary to my item 4 of the initial post of this PR I did make your change to the index.html. I think your call is correct about keeping it in the sub folder considering the page is not a single page but a section with linked pages in the notes folder. If it were just a single page it might make sense to keep it as a notes.html page in the root of the repository.

This PR fixes an issue with the current site, and brings back the Schwab page. Would be nice to get the second approval and get this live on the site

**Review by Carlgo11 — APPROVED** [review]
Nice work! 👍

**Pull request merged**
